service/popm/popm.go

```diff
@@ -253,6 +279,33 @@ func createTx(l2Keystone *hemi.L2Keystone, btcHeight uint64, utxo *bfgapi.Bitcoi
 	return &btx, nil
 }
 
+func (m *Miner) shouldMineKeystone(ctx context.Context, ks *hemi.L2Keystone) (bool, error) {
+	serialized := hemi.L2KeystoneAbbreviate(*ks).Serialize()
+	response, err := m.callBFG(ctx, 5*time.Second, &bfgapi.PopTxsForL2BlockRequest{
```

Comment:
we can't do this from popm; it is on a different listener...find a different way